usage: expand and clean up keybindings

josephholsten opened the pull request:
I've simplified some of the wording inspired by nvi's very concise descriptions.

josephholsten commented:
hrm, github's markdown processor is putting `<p>` tags around some of the list elements for reasons that aren't clear to me. This definitely undermines the layout I was attempting

driusan commented on USAGE.md:
```diff
@@ -17,39 +17,46 @@ position, like vi. Unlike vi, holding CTRL and typing a movement command
+-   `h`: move left by columns
+-   `j`: move down by lines
+-   `k`: move up by lines
+-   `l`: move right by columns
```
I find the original wording clearer here (though it should probably say "*count* chars" instead of "one char" to be consistent with the delete wording and be more accurate.)

driusan commented on USAGE.md:
```diff
@@ -17,39 +17,46 @@ position, like vi. Unlike vi, holding CTRL and typing a movement command
+-   `w`: move to next word
+-   `b`: move back word
+-   `$`: move to last column
+-   `^`: move to first character
```
`w` and `b` can have count modifiers too, and I still find "start of line" and "end of line" easier to understand.

driusan commented on USAGE.md:
```diff
@@ -17,39 +17,46 @@ position, like vi. Unlike vi, holding CTRL and typing a movement command
--   `x`: delete current char
--   Backspace: delete durrently selected text (or previous character if
-    nothing is selected)
+-   `x`: delete character
```
technically, this can delete more than 1 character if something is selected (and it can have a count modifier)

driusan commented on USAGE.md:
```diff
@@ -17,39 +17,46 @@ position, like vi. Unlike vi, holding CTRL and typing a movement command
--   Up: scroll viewport up half a screen
--   Down: scroll viewport down half a screen
--   Esc: Save if buffer has been modified, Exit if buffer has not been
+-   `p`: paste
```
the `u` for Undo disappeared, and Paste is a builtin-command (but paste isn't), so it's better to keep it capitalized so that the word is executable in the documentation.

josephholsten replied:
did it really disappear? it seems to work on my latest build

driusan replied:
I mean it disappeared from the documentation in your patch

driusan commented on USAGE.md:
```diff
@@ -74,6 +83,26 @@ a forward or backwards movement command.)
+-   `^`: delete to start of line
+-   `G`: delete to line, or end of file with no prefix
+
+-   `d`: delete entire line
```
this can have a count modifier too.

josephholsten commented:
@driusan I've updated this, fixing all your issues. I also emboldened all **Command** keywords, and removed the doc for specific commands from usage, instead pointing people toward the `COMMANDS.md` file
